java/client/src/org/openqa/selenium/support/ui/Select.java

```diff
@@ -59,20 +59,23 @@ public Select(WebElement element) {
    * @return Whether this select element support selecting multiple options at the same time? This
    *         is done by checking the value of the "multiple" attribute.
    */
+  @Override
```

Comment:
can you revert all the Overrides added? it's not required for implementing interfaces, only overriding methods extended.